4.7 mobile nav intro (#126)
smartdevicelink/sdl_android_guides: docs/Mobile Navigation/Introduction/index.md

The basic connection is the similar for all apps. Please follow [Getting Started > Integration Basics](Getting Started/Integration Basics) for more information.

The first difference for a navigation app is the `appHMIType` of `NAVIGATION` that has to be set in the creation of the `SdlManager`. Navigation apps are also non-media apps.

> The second difference is a property called `setSdlSecurity()` that needs to be set in the `SdlManager.Builder ` if connecting to a version of Core that requires secure video & audio streaming. This property requires an array of classes of Security Managers, which will extend the `SdlSecurityBase` class. These security libraries are provided by the OEMs themselves, and will only work for that OEM. There is not a general catch-all security library.

Reviewer: ```suggestion
The second difference is the requirement to call the `setSdlSecurity(List<Class<? extends SdlSecurityBase>> secList)` method from the `SdlManager.Builder ` if connecting to an implementation of Core that requires secure video & audio streaming. This method requires an array of Security Managers, which will extend the `SdlSecurityBase` class. These security libraries are provided by the OEMs themselves, and will only work for that OEM. There is not a general catch-all security library.
```
